OpenRA.Game/Traits/Player/FrozenActorLayer.cs

```diff
@@ -64,6 +64,18 @@ public FrozenActor(Actor self, PPos[] footprint, Shroud shroud, bool startsRevea
 				.Where(m => shroud.Contains(m))
 				.ToArray();
 
+			if (Footprint.Length == 0)
+				throw new ArgumentException(("This frozen actor has no footprint! Please report a bug!\n" +
+					"Actor Name: {0}\nActor Location: {1}\n" +
```

Comment:
Consistency nit-pick: The location string should go on its own line as another concat.